docs/configuration.md

```diff
@@ -0,0 +1,297 @@
+  which bootstrap stakeholders should be used, what should be total
+  balance, etc. When node is launched with spec as genesis, it
+  automatically constructs genesis data from it. This way to specify
+  genesis is supposed to be used for everything except
+  mainnet. Example:
+
+```
+    genesis:
+      spec:
+        initializer:
+          testnetInitializer:
+            testBalance:
+              poors:        12
+              richmen:      4
+              richmenShare: 0.99
+              useHDAddresses: True
+              totalBalance: 600000000000000000
+            fakeAvvmBalance:
+              count: 10
+              oneBalance: 100000
+            distribution:
+              testnetRichmenStakeDistr: []
+            seed: 0
+        blockVersionData:
+          scriptVersion:     0
+          slotDuration:      7000
+          maxBlockSize:      2000000
+          maxHeaderSize:     2000000
+          maxTxSize:         4096 # 4 Kb
+          maxProposalSize:   700 # 700 bytes
+          mpcThd:            0.01 # 1% of stake
+          heavyDelThd:       0.005 # 0.5% of stake
+          updateVoteThd:     0.001 # 0.1% of total stake
+          updateProposalThd: 0.1 # 10% of total stake
+          updateImplicit:    10 # slots
+          softforkRule:
+            initThd:        0.9 # 90% of total stake
+            minThd:         0.6 # 60% of total stake
+            thdDecrement:   0.05 # 5% of total stake
+          txFeePolicy:
+            txSizeLinear:
+              a: 155381 # absolute minimal fees per transaction
+              b: 43.946 # additional minimal fees per byte of transaction size
+          unlockStakeEpoch: 18446744073709551615 # last epoch (maxBound @Word64)
+        protocolConstants: &dev_core_genesis_spec_protocolConstants
+          k: 2
+          protocolMagic: 55550001
+          vssMinTTL: 2
+          vssMaxTTL: 6
+        ftsSeed: "c2tvdm9yb2RhIEdndXJkYSBib3JvZGEgcHJvdm9kYSA="
+        heavyDelegation: {}
+        avvmDistr: {}
+```
+
+### Initializer
+
+There are two substantially different types of genesis spec, with the
+difference in `initializer` value. Initializer can be either
+`testnetInitializer` or `mainnetInitializer`.
+
+The key point of `testnetInitializer` is that it contains seed which
+is used to generate secret keys. It's quite convenient for testnet,
+because there we want everyone to be able to have some coins. So we
+can deterministically generate a lot of addresses with some coins and
+let people use them. It's also convenient for devnet clusters
+(i. e. clusters primarily used by developers to test something),
+because in this case we also want to know all secrets. An example of
+`testnetInitializer` was provided above. The most interesting part
+there is `distribution` field:
+
+```
+  distribution:
+   testnetRichmenStakeDistr: []
+```
+
+It means that generated richmen will also be bootstrap stakeholders
+and will be participating in SSC. Note that richmen's keys are
```

Comment:
The reference to SSC begs a reference to the background documentation, where one could self-educate.

Reply:
I am not sure if we have any (up-to-date) documentation, I will add a link if I find it.

Reply:
Perhaps including "Shared Seed Computation" and linking to https://cardanodocs.com/technical/blocks/. I didn't recall what SSC referred to either.

Reply:
@gromakovsky, would you mind adding this reference -- and, perhaps a bit expand on its significance in this particular context?

Reply:
I don't think that link is a good source of information. Instead, I wrote a short explanation in Prerequisites:
```
SSC is Shared Seed Computation, it's performed by rich nodes (with big stake) to agree upon the seed which will be used for leaders selection for an epoch.
```